### ISSUE TITLE
v1.2.4, missing directories that Windows 7 shows

### ISSUE THREAD

**Issue by mandree** [relan/exfat#42]
related: dorimanx/exfat-nofuse/issues/87 (but different results)

I have an SDXC card where a Cygwin "find" under Windows 7 shows many more directories than Linux's find with exfat-fuse v1.2.4. exfatfsck does not pass, but aborts (separate issue), yet exfat-fuse sees more files than exfat-nofuse in the same directory.

diff -u between Windows's/Cygwin's "find" and exfat on Linux:

```
--- filelist-windows7.txt   2016-07-18 23:26:13.099195140 +0200
+++ filelist-exfat-fuse.txt 2016-07-18 22:53:32.343360700 +0200
@@ -2413,33 +2413,6 @@
 ./DCIM/12760612/DSC02565.JPG
 ./DCIM/12760612/DSC02566.ARW
 ./DCIM/12760612/DSC02566.JPG
-./DCIM/12960617
-./DCIM/12960617/DSC02578.ARW
[...]
-./DCIM/12960617/DSC02590.ARW
-./DCIM/12960617/DSC02590.JPG
 ./DCIM/13160619
 ./DCIM/13160619/DSC02605.ARW
 ./DCIM/13160619/DSC02605.ARW.xmp
@@ -2974,296 +2947,6 @@
 ./DCIM/13560709/DSC02941.ARW
 ./DCIM/13560709/DSC02941.JPG
 ./DCIM/13660710
-./DCIM/13760715
-./DCIM/13760715/DSC02945.ARW
-./DCIM/13760715/DSC02945.JPG
[...]
-./DCIM/13960716/DSC03076.JPG
-./DCIM/13960716/DSC03077.ARW
-./DCIM/13960716/DSC03077.JPG
-./DCIM/13960716/DSC03078.ARW
-./DCIM/13960716/DSC03078.JPG
-./MP_ROOT
-./MP_ROOT/100ANV01
-./MP_ROOT/101ANV01
-./MP_ROOT/102ANV01
-./MP_ROOT/103ANV01
-./MP_ROOT/104ANV01
[...]
-./MP_ROOT/137ANV01
-./MP_ROOT/138ANV01
-./MP_ROOT/139ANV01
 ./PRIVATE
 ./PRIVATE/AVCHD
 ./PRIVATE/AVCHD/BDMV
```

**Comment by relan** [comment]
Probably related to #41. I'd like to investigate it first. Is this the same SD card with the same content?

**Comment by mandree** [comment]
Yes, it is.

**Comment by mandree** [comment]
Coincidentally also fixed in v1.2.5.

**Comment by relan** [comment]
That's unexpected. Thanks for checking!
